scripts/calculateRewards/proportionalPrizeContest.ts

```diff
@@ -47,3 +47,50 @@ export function calculateProportionalPrizeContest({
 
   return rewardsPerReferrer
 }
+
+export function calculateSqrtProportionalPrizeContest({
+  kpiData,
+  rewards,
+}: {
+  kpiData: KpiRow[]
+  rewards: BigNumber
+}) {
+  const referrerKpis = kpiData.reduce(
+    (acc, row) => {
+      if (!(row.referrerId in acc)) {
+        acc[row.referrerId] = BigInt(row.kpi)
+      } else {
+        acc[row.referrerId] += BigInt(row.kpi)
+      }
+      return acc
+    },
+    {} as Record<string, bigint>,
+  )
+
+  const referrerPowerKpis = Object.entries(referrerKpis).reduce(
+    (acc, [referrerId, kpi]) => {
+      acc[referrerId] = BigNumber(kpi).sqrt()
```

Comment:
random thought, is it likely we'd want to play with the power? i wonder if we should pass in a number (0.5 for sqrt) and use `.pow` since the variable name mentions power 🤷🏻‍♀️